app/helpers/data_lists_helper.rb

```diff
@@ -67,16 +67,11 @@ def dl_linked_version(description)
     end
     return new_words.join(" ")
   end
-  
-end
 
+  def make_indentation(n)
+    ## HTML string below needs to keep internal quotes as single, because it gets embedded in JSON
```

Comment:
We could still use this helper in the view, but returning raw HTML in as a JSON value is usually frowned upon.